upgrade/scripts/sls/csm_can_to_chn/sls_updates.py

```diff
@@ -87,6 +98,22 @@ def create_bican_network(networks, default_route_network_name):
         bican = BicanNetwork(default_route_network_name=default_route_network_name)
         networks.update({bican.name(): bican})
 
+def delete_can_network(networks):
+    """Delete SLS CAN network data structure."""
+
+    if networks.get("CHN") is None:
+        click.secho("CHN subnet not found in SLS", fg="red")
+        sys.exit(1)
+
+    if networks.get("BICAN") is not None:
+        default_route = networks.get("BICAN").system_default_route()
+        if default_route != "CHN":
+            click.secho(f"The current system default route is set to {default_route}!! \nThis script should only be used when the default route is set to CHN", fg="red")
```

Comment:
Better messaging.  `The current user network is set to {default_route}.  To remove the CAN this value must be set to CHN first.`